##Parameter for collision_avoidance.cpp

##  期望位置_x;
target_x : 4

## 期望位置_y
target_y : -0.3

##追踪部分位置环P
p_xy : 1

## 单位:米
R_outside : 1.3

## 单位:米
R_inside : 0.3

## 追踪部分速度限幅
vel_track_max : 0.4

## 大圈比例系数
p_R : 0.6

## 小圈比例系数
p_r : 1

## 避障部分最大速度
vel_collision_max : 0.5

## 总速度最大速度
vel_sp_max : 0.4


## 激光雷达范围设定
range_min : 45
range_max : 315
